lib/themes/reps/css/reps.css.less

```diff
@@ -51,6 +50,21 @@
 
 /* end REPR-1305 */
 
+/* REPR-1673 body images responsive */ 
```

Comment:
Can you review the indentation?
Just for the next one who have to work in the file.

In advance thanks